packages/metals-vscode/package.json

```diff
@@ -357,6 +357,12 @@
           ],
           "markdownDescription": "Specifies which UI should be used for tests. `Code lenses` shows virtual text above test class, just like it was previously. `Test explorer` is a new option which shows gutter icons on the left of the file in a less intrusive way. Moreover, this option comes with a new panel called `Testing` which lists all test suites in your codebase and allows you to browse tests and navigate to the file which contains them."
         },
+        "metals.customProjectRoot": {
+          "type": "string",
+          "default": "",
+          "scope": "machine-overridable",
```

Comment:
Why this scope?

Reply:
We need a scope that will allow for the setting to be available on the workspace folder level. This one seems most reasonable: https://code.visualstudio.com/api/references/contribution-points#contributes.configuration.

Reply:
Thanks.